addons/hr_work_entry_holidays/models/hr_contract.py

```diff
@@ -73,7 +73,7 @@ def _get_interval_leave_work_entry_type(self, interval, leaves, bypassing_codes)
     def _get_leave_domain(self, start_dt, end_dt):
         # Complete override, compare over holiday_id.employee_id instead of calendar_id
         return [
-            ('time_type', '=', 'leave'),
+            ('time_type', 'in', ['leave', 'other']),
```

Comment:
Hi @wtaferner 

Thanks for the feedback :)

Why not removing the whole chunk ? ``time_type`` can only be of type leave or other. I suppose an empty value is ok too.

```diff
-             ('time_type', '=', 'leave'),
```

Reply:
As always trying to do the minimal diff to get a review, but as you asked for it, you now get the full package. I hope you like it 😜